whylog/constraints/__init__.py

```diff
@@ -194,16 +194,20 @@ class IdenticalConstraint(AbstractConstraint):
 
     TYPE = ConstraintType.IDENTICAL
 
+    PARAM_VALUE = "value"
```

Comment:
Following PARAMS should be:
PARAMS = [PARAM_VALUE]
otherwise user wouldn't know that this param exists.

Convention in this file is to write 'value' rather than "value"